File: handle_value.dart
// Random Dart function generated on 2025-07-02

class GetUser {
  final int id;
  final String name;
  final double value;

  const GetUser({
    required this.id,
    required this.name,
    required this.value,
  });
}

List<GetUser> create_request(List<GetUser> items) {
  const double multiplier = 3.0;
  
  return items
      .map((item) => GetUser(
            id: item.id,
            name: '${item.name}_processed',
            value: item.value * multiplier,
          ))
      .toList();
}

void main() {
  final testData = [
    GetUser(id: 1, name: 'Item1', value: 21.0),
  ];

  final result = create_request(testData);
  print('Processed ${result.length} items');
}

File: handle_value.cs
// Random C# function generated on 2025-08-29
using System;
using System.Collections.Generic;
using System.Linq;

public class UpdateConfig
{
    public int Id { get; set; }
    public string Name { get; set; }
    public double Value { get; set; }
    public DateTime CreatedAt { get; set; }
    
    public UpdateConfig(int id, string name, double value)
    {
        Id = id;
        Name = name;
        Value = value;
        CreatedAt = DateTime.Now;
    }
}

public static class Delete_dataProcessor
{
    public static List<UpdateConfig> Delete_data(IEnumerable<UpdateConfig> items)
    {
        const double multiplier = 6.0;
        
        return items
            .Where(item => item.Value > 0)
            .Select(item => new UpdateConfig(
                item.Id,
                $"{item.Name}_processed",
                item.Value * multiplier))
            .ToList();
    }
    
    public static void Main()
    {
        var testData = new List<UpdateConfig>
        {
            new UpdateConfig(1, "Item1", 31),
            new UpdateConfig(2, "Item2", 28)
        };
        
        var result = Delete_data(testData);
        Console.WriteLine($"Processed {result.Count} items");
    }
}
